pkg/build/api/install/apigroup.go

```diff
@@ -0,0 +1,26 @@
+package install
+
+import (
+	"k8s.io/kubernetes/pkg/apimachinery/announced"
+
+	"github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/pkg/build/api/v1"
+)
+
+const importPrefix = "github.com/openshift/origin/pkg/build/api"
```

Comment:
This may end up being a two stage process.  As I recall, some of the client generators freak out when the last step of the package isn't the group